packages/engine/src/physics/systems/PhysicsSystem.ts

```diff
@@ -57,6 +59,15 @@ export const PhysicsPrefabs = {
   collider: 'collider' as const
 }
 
+export function smoothKinematicBody(entity: Entity, alpha: number) {
+  const rigidbodyComponent = getComponent(entity, RigidBodyComponent)
+  const transformComponent = getComponent(entity, TransformComponent)
+  rigidbodyComponent.position.lerp(transformComponent.position, alpha)
+  rigidbodyComponent.rotation.slerp(transformComponent.rotation, alpha)
```

Comment:
I believe we are using lerp elsewhere for rotations, and slerp is quite expensive. Perhaps when we need, we can add an option to slerp per rigidbody, but for most cases interpolating between frames a basic lerp should suffice.